scripts/KafkaServer.py

```diff
@@ -175,25 +176,32 @@ def writeKafkaConfig (self):
         keyCertPasswords = ("ssl.truststore.password=%s\nssl.keystore.password=%s\nssl.key.password="
                             "%s\n") % (self.pwd, self.pwd, self.pwd)
         addr = socket.gethostbyname(socket.getfqdn())
-        al   = "advertised.listeners"
+        alStr = None
+        if (self.al != None):
```

Comment:
This code will almost certainly work, but it's worth mentioning that `== None` can have very surprising behavior in many cases. `None` is a singleton, so the right way to do this is with `is`, like `self.al is not None` / `alStr is None`. This is a [part of PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations), which is the manifesto on writing Python the Right Way